src/PermissionServiceProvider.php

```diff
@@ -67,13 +71,17 @@ protected function offerPublishing()
 
     protected function registerCommands()
     {
-        $this->commands([
+        if (! $this->app->runningInConsole()) {
+            return;
+        }
```

Comment:
Will this block the ability of an application (such as via the app's Admin interface) from calling any of these Artisan commands? 

I recall reverting a similar PR quite awhile back for something akin to this sort of change. Not sure if it was exactly this though.

Reply:
> Will this block the ability of an application from calling any of these Artisan commands?

yes
> Not sure if it was exactly this though

https://github.com/spatie/laravel-permission/commit/6b1aedd0f25dfbca19c174b9f03bf2e69d9eeb28, it was because they want to use cache reset from the app in Laravel Nova, but maybe just leave that command, I see it useful in a interface app to create roles, permissions by command

reverted